### PR TITLE
added the pr template

### LINKED ISSUE #728: Add PR template
Add PR template.

### PR DESCRIPTION
**Pull request by swastik959**
Fixes #728

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+<!--
+In a couple sentences, explain why this PR is needed and what it addresses. This should be an explanation a non-developer user can understand and covers the "why" question. It should also clearly indicate whether this PR represents an addition, a change, or a fix of existing behavior. This explanation will be used to assist in the release note drafting process.
+
+THIS IS MANDATORY.
+-->
+
+## Related issue
+
+<!--
+Please link the GitHub issue this pull request resolves in the format of `Closes #1234`. If you discussed this change
```
Comment (eddycharly) [inline .github/PULL_REQUEST_TEMPLATE.md:12]:
```suggestion
Please link the GitHub issue this pull request resolves in the format of `Fixes #1234`. If you discussed this change
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+with a maintainer, please mention her/him using the `@` syntax (e.g. `@JimBugwadia`).
```
Comment (eddycharly) [inline .github/PULL_REQUEST_TEMPLATE.md:13]:
```suggestion
with a maintainer, please mention her/him using the `@` syntax (e.g. `@eddycharly`).
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+If this change neither resolves an existing issue nor has sign-off from one of the maintainers, there is a
+chance substantial changes will be requested or that the changes will be rejected.
+
+You can discuss changes with maintainers in the [Kyverno Slack Channel](https://kubernetes.slack.com/).
```
Comment (eddycharly) [inline .github/PULL_REQUEST_TEMPLATE.md:18]:
```suggestion
You can discuss changes with maintainers in the [kyverno-chainsaw Slack Channel](https://kubernetes.slack.com/).
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+-->
+
+## Milestone of this PR
+<!--
+
+Add the milestone label by commenting `/milestone 1.2.3`.
+
+-->
+
```
Comment (eddycharly) [inline .github/PULL_REQUEST_TEMPLATE.md:27]: you can remove that we don't set milestone on PRs

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+>
+> /kind api-change
+> /kind bug
+> /kind cleanup
+> /kind design
+> /kind documentation
+> /kind failing-test
+> /kind feature
+-->
+
```
Comment (eddycharly) [inline .github/PULL_REQUEST_TEMPLATE.md:42]: remove that, we don't use it

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: roles-dictionary
+  namespace: default
+data:
+  allowed-roles: "[\"cluster-admin\", \"cluster-operator\", \"tenant-admin\"]"
+```
+-->
+
```
Comment (eddycharly) [inline .github/PULL_REQUEST_TEMPLATE.md:69]: remove that, we don't use it

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+<!--
+Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of
+them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
+-->
+
+- [ ] I have read the [contributing guidelines](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md).
+- [ ] I have read the [PR documentation guide](https://github.com/kyverno/kyverno/blob/main/.github/pr_documentation.md) and followed the process including adding proof manifests to this PR.
+- [ ] This is a bug fix and I have added unit tests that prove my fix is effective.
+- [ ] This is a feature and I have added CLI tests that are applicable.
+- [ ] My PR needs to be cherry picked to a specific release branch which is <replace>.
```
Comment (eddycharly) [inline .github/PULL_REQUEST_TEMPLATE.md:81]:
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+<!--
+Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of
+them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
+-->
+
+- [ ] I have read the [contributing guidelines](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md).
+- [ ] I have read the [PR documentation guide](https://github.com/kyverno/kyverno/blob/main/.github/pr_documentation.md) and followed the process including adding proof manifests to this PR.
+- [ ] This is a bug fix and I have added unit tests that prove my fix is effective.
+- [ ] This is a feature and I have added CLI tests that are applicable.
```
Comment (eddycharly) [inline .github/PULL_REQUEST_TEMPLATE.md:80]:
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+
+## Checklist
+
+<!--
+Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of
+them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
+-->
+
+- [ ] I have read the [contributing guidelines](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md).
+- [ ] I have read the [PR documentation guide](https://github.com/kyverno/kyverno/blob/main/.github/pr_documentation.md) and followed the process including adding proof manifests to this PR.
```
Comment (eddycharly) [inline .github/PULL_REQUEST_TEMPLATE.md:78]: are those links the correct ones ?

Comment (swastik959) [reply .github/PULL_REQUEST_TEMPLATE.md:40]: yes I guess so

Comment (eddycharly) [reply .github/PULL_REQUEST_TEMPLATE.md:40]: Ok, we are moving things in the `kyverno/community` repo but it's not ready yet. I will update the templates later.

**Comment by eddycharly** [comment]
LGTM, thanks!

**Review by eddycharly — APPROVED** [review]

**Pull request merged**